packages/aws-cdk-lib/aws-synthetics/lib/runtime.ts

```diff
@@ -22,12 +22,191 @@ export enum RuntimeFamily {
  * Runtime options for a canary
  */
 export class Runtime {
+
+  /**
+   * `syn-1.0` includes the following:
+   * - Lambda runtime Node.js 10.x
+   * - Puppeteer-core version 1.14.0
+   * - Chromium version that matches Puppeteer-core 1.14.0
+   *
+   * @see https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Synthetics_Library_nodejs_puppeteer.html#CloudWatch_Synthetics_runtimeversion-1.0
+   * @deprecated Legacy runtime no longer supported by AWS Lambda. Migrate to the latest NodeJS Puppeteer runtime.
+   */
+  public static readonly SYNTHETICS_NODEJS_PUPPETEER_1_0 = new Runtime('syn-1.0', RuntimeFamily.NODEJS);
```

Comment:
I don't think we should add ones that are already deprecated. Can we remove these?

Reply:
My original thought for all these PRs was to have parity between the SDK and the CDK, as a starting off point for continuous, automatic checks. I can always ignore SDK versions/engines/runtimes etc. that are deprecated

Reply:
Done 👍